### ISSUE TITLE
Incorrectly reporting issue with DynamoDB table Read and Write Capacity units

### ISSUE THREAD

**Issue by reidca** [awslabs/cfn-python-lint#147]
```
E3002 Property Resources/ConfigTable/Properties/ProvisionedThroughput/ReadCapacityUnits should be of type Integer
cloudformation/ec2/management/templates/instance-scheduler.yaml:945:9

```
```
E3002 Property Resources/ConfigTable/Properties/ProvisionedThroughput/WriteCapacityUnits should be of type Integer
cloudformation/ec2/management/templates/instance-scheduler.yaml:946:9
```

Resource is:

```
ConfigTable:
    Type: 'AWS::DynamoDB::Table'
    Properties:
      AttributeDefinitions:
        - AttributeName: type
          AttributeType: S
        - AttributeName: name
          AttributeType: S
      KeySchema:
        - AttributeName: type
          KeyType: HASH
        - AttributeName: name
          KeyType: RANGE
      ProvisionedThroughput:
        ReadCapacityUnits: 2
        WriteCapacityUnits: 2
```

**Comment by kddejong** [comment]
@reidca I'm unable to replicate this.  Can you get me the version of cfn-lint and python you are using?

**Comment by reidca** [comment]
I have just updated this using pip3 as I was having issues using Pip on my MAC, hopefully I now have the latest version.


cfn-lint --version
cfn-lint 0.0.8

I have ran the linter again and the same errors are reported:

```
Carl-Reid-MacBook-Pro:aws carl.reid$ cfn-lint --template /Users/carl.reid/Source/aws/cloudformation/ec2/management/templates/instance-scheduler.yaml
E2530 Type for Parameter should be Integer, MinValue should be at least 128, and MaxValue equal or less than 1536 at Parameters/MemorySize/Type
/Users/carl.reid/Source/aws/cloudformation/ec2/management/templates/instance-scheduler.yaml:20:5

E3002 Property Resources/ConfigTable/Properties/ProvisionedThroughput/ReadCapacityUnits should be of type Integer
/Users/carl.reid/Source/aws/cloudformation/ec2/management/templates/instance-scheduler.yaml:945:9

E3002 Property Resources/ConfigTable/Properties/ProvisionedThroughput/WriteCapacityUnits should be of type Integer
/Users/carl.reid/Source/aws/cloudformation/ec2/management/templates/instance-scheduler.yaml:946:9
```

**Comment by fatbasstard** [comment]
Can you upgrade to the latest version (`0.3.0`) and try again. There have been quite some improvements since version `0.0.8`

**Comment by reidca** [comment]
I have had lots of problems upgrading with Pip. I seem to have managed to get this working now by specifying the --user option. The version is now 0.3.0 and this problem seems to have been resolved. Sorry for raising the issue incorrectly.
